doc/user/content/concepts/clusters.md

```diff
@@ -28,8 +28,15 @@ to be associated with a cluster:
 You choose the size of your cluster (`25cc`, `50cc`, `100cc`, etc.) based on
 the resource requirements of your workload. Larger clusters have more compute
 resources available and can therefore process data faster and handle larger data
-volumes. You can resize a cluster at any time, even while the cluster is
-running, to respond to changes in your workload.
+volumes.
+
+You can resize a cluster to respond to changes in your workload:
+
+- For clusters that contain storage objects (i.e., sources and sinks), resizing
+  incurs downtime.
+
+- For clusters that contain compute objects only (i.e., indexes and materialized
+  views), resizing does not incur downtime.
```

Comment:
There's downtime in both cases. At least until [graceful cluster configuration](https://github.com/MaterializeInc/materialize/issues/20010) ships.

Reply:
Got it. 👍  
I'll just update to state that resizing incurs downtime on this page.  On the billing page, then, I won't mention that, but will link to this page.